index.html

```diff
@@ -28,6 +29,146 @@ <h1 class="headline-title">I’m Emre<br>Glad to see you!</h1>
+                <p class="card-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tags">
+                    <li class="tag">html</li>
+                    <li class="tag">css</li>
+                    <li class="tag">javaScript</li>
+                </ul>
+                <button class="btn card-button" type="button">See Project</button>
+            </div>
+        </div>
+
+        <div class="works-card">
+            <div class="card-img card3">
+            </div>
+            <div class="card-text">
+                <h2 class="card-title">Tonic</h2>
+                <ul class="card-details">
+                    <li class="company">CANOPY</li>
+                    <li class="circle"><img src="img/circle.png" alt="circle"></li>
+                    <li class="job">Back End Dev</li>
+                    <li class="circle"><img src="img/circle.png" alt="circle"></li>
+                    <li class="year">2015</li>
+                </ul>
+                <p class="card-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tags">
+                    <li class="tag">html</li>
+                    <li class="tag">css</li>
+                    <li class="tag">javaScript</li>
+                </ul>
+                <button class="btn card-button" type="button">See Project</button>
+            </div>
+        </div>
+
+        <div class="works-card">
+            <div class="card-img card4">
+            </div>
+            <div class="card-text">
+                <h2 class="card-title">Multi-Post
+Stories</h2>
+                <ul class="card-details">
+                    <li class="company">CANOPY</li>
+                    <li class="circle"><img src="img/circle.png" alt="circle"></li>
+                    <li class="job">Back End Dev</li>
+                    <li class="circle"><img src="img/circle.png" alt="circle"></li>
+                    <li class="year">2015</li>
+                </ul>
+                <p class="card-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tags">
+                    <li class="tag">html</li>
+                    <li class="tag">css</li>
+                    <li class="tag">javaScript</li>
+                </ul>
+                <button class="btn card-button" type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-me">
+        <div class="about-me-top">
+            <h2 class="about-me-title">About<br>Myself</h2>
+            <p class="about-me-info">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="about-me-link" href="#">LET'S CONNECT</a>
```

Comment:
- [ ] I kindly suggest that you use any of the header tag from `h2` to `h6` for "LET'S CONNECT" because it's a header.